pom.xml

```diff
@@ -39,7 +39,7 @@
 
 
     <properties>
-        <revision>1.1.1</revision>
+        <revision>1.3.0-SNAPSHOT</revision>
```

Comment:
Shouldn't the new version be 1.2.1 instead of 1.3.0 (latest released version is 1.2.0 and for some reason differs from version defined in pom)? There should not be any new or changed funtionality so I wouldn't clasify it as minor change.

Reply:
Following [this PR](https://github.com/admin-shell-io/java-serializer/pull/61) it should be 1.2.1-SNAPSHOT. I really don't have a strong opinion on this but we should make sure it's consistent.